### ISSUE TITLE
Trying to get dataset to work for RVQ-VAE training

### ISSUE THREAD

**Issue by HoustonHuff** [andypinxinliu/GestureLSM#16]
Hello, thank you very much for your work on this project.

I've been having trouble getting the LMDB dataset loading going for the RVQ-VAE training; in particular I'm wanting to focus on the training of the face and am taking this training execution from train_rvq.sh:

`python rvq_beatx_train.py --batch-size 256 --lr 2e-4 --total-iter 300000 --lr-scheduler 200000 --nb-code 1024 --width 512 --code-dim 128 --down-t 2 --depth 3 --dilation-growth-rate 3 --out-dir outputs/rvqvae_simple --vq-act relu --quantizer ema_reset --loss-vel 0.5 --recons-loss l1_smooth --exp-name RVQVAE --body_part face`

However, I'm running into an LMDB loading error:

`lmdb.Error: .../GestureLSM/datasets/beat_cache/beat_smplx_en_emage_2_rvqvae/train/smplxflame_30_cache: No such file or directory`

My understanding of LMDB is that if it's missing necessary files like the train.txt and test.txt for the specific LMDB file structure it'll say the whole directory can't be found, and sure enough, even after re-organizing the beat_v2.0.0 dataset directory to follow the path the code's looking for, I'm still getting this error, or similar errors asking for files within smplxflame_30_cache that are expected by LMDB but not present; I have another LMDB dataset from a different project that I'm cross-referencing and it indeed seems to have file structure differences.

The file structure I'm looking at with BEAT is:

beat_english_v2.0.0
----sem
--------1_wayne_0_1_1.txt
--------...
----smplxflame_30
--------1_wayne_0_1_1.npz
--------...
----textgrid
--------1_wayne_0_1_1.TextGrid
--------...
----wave16k
--------1_wayne_0_1_1,wav
--------...
----weights
--------AESKConv_240_100.bin
--------vocab.pkl
----readme.md
----train_test_split.csv

Any guidance on how to get things working with the dataset would be greatly appreciated.
